src/services/mongoose/wallet-on-chain-addresses.ts

```diff
@@ -1,21 +1,21 @@
-import { User } from "@services/mongoose/schema"
 import {
   CouldNotFindError,
   PersistError,
   RepositoryError,
   UnknownRepositoryError,
 } from "@domain/errors"
 import { baseLogger } from "@services/logger"
+import { Wallet } from "@services/mongoose/schema"
 
 export const WalletOnChainAddressesRepository = (): IWalletOnChainAddressesRepository => {
   const persistNew = async (
-    walletId: WalletId,
+    id: WalletId,
```

Comment:
we need to update this to one param rule

Reply:
I don't understand this comment?

Reply:
He means single object arg

```ts
const persistNew = async ({
  id,
  onChainAddress,
}: {
  id: WalletId
  onChainAddress: OnChainAddressIdentifier
}): Promise<OnChainAddressIdentifier | RepositoryError> => {
```